File: 4. Median of Two Sorted Arrays/Solution.py
class Solution(object):
    def findMedianSortedArrays(self, nums1, nums2):
        """
        :type nums1: List[int]
        :type nums2: List[int]
        :rtype: float
        """
        nums = sorted(nums1 + nums2)
        if len(nums) % 2 == 1:
            return nums[int(len(nums) / 2)]
        else:
            med1 = nums[int(len(nums) / 2) - 1]
            med2 = nums[int(len(nums) / 2)]
            return (med1 + med2) / 2.0

File: 4. Median of Two Sorted Arrays/Solution.cs
public class Solution {
    public double FindMedianSortedArrays(int[] nums1, int[] nums2)
    {
        int[] nums = nums1.Concat(nums2).ToArray();
        Array.Sort(nums);
        if (nums.Length % 2 == 1)
            return nums[nums.Length / 2];
        else
            return (nums[nums.Length / 2 - 1] + nums[nums.Length / 2]) / 2.0;
    }
}